components/icons/SearchIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+
+interface SearchIconProps {
+  color: string;
+}
+
+const SearchIcon: FC<SearchIconProps> = ({ color }) => {
+  return (
+    <svg
+      width="20"
+      height="20"
+      viewBox="0 0 20 20"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```

Comment:
ok these are actually causing some console errors—when react deals with SVGs it doesn't like snakecase. changing these to camelcase should fix the problem! 

```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```

<img width="660" alt="Screen Shot 2023-03-06 at 10 21 00 PM" src="https://user-images.githubusercontent.com/62641231/223319901-d13b5b02-d966-4849-84eb-72d7f5f77470.png">